<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="7" passed="6" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-12-07T13:10:23 IST" name="Default suite" finished-at="2022-12-07T13:10:32 IST" duration-ms="8801">
    <groups>
    </groups>
    <test started-at="2022-12-07T13:10:23 IST" name="Default test" finished-at="2022-12-07T13:10:32 IST" duration-ms="8801">
      <class name="TestTopics.AssertsTopics">
        <test-method is-config="true" signature="setup()[pri:0, instance:TestTopics.AssertsTopics@6aba2b86]" started-at="2022-12-07T13:10:23 IST" name="setup" finished-at="2022-12-07T13:10:29 IST" duration-ms="6522" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="test()[pri:1, instance:TestTopics.AssertsTopics@6aba2b86]" started-at="2022-12-07T13:10:29 IST" name="test" finished-at="2022-12-07T13:10:29 IST" duration-ms="12" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method signature="testForNotEqual()[pri:2, instance:TestTopics.AssertsTopics@6aba2b86]" started-at="2022-12-07T13:10:29 IST" name="testForNotEqual" finished-at="2022-12-07T13:10:29 IST" duration-ms="5" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForNotEqual -->
        <test-method signature="testForTrue()[pri:3, instance:TestTopics.AssertsTopics@6aba2b86]" started-at="2022-12-07T13:10:29 IST" name="testForTrue" finished-at="2022-12-07T13:10:29 IST" duration-ms="6" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForTrue -->
        <test-method signature="testForFalse()[pri:4, instance:TestTopics.AssertsTopics@6aba2b86]" started-at="2022-12-07T13:10:29 IST" name="testForFalse" finished-at="2022-12-07T13:10:29 IST" duration-ms="17" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForFalse -->
        <test-method signature="testForNull()[pri:5, instance:TestTopics.AssertsTopics@6aba2b86]" started-at="2022-12-07T13:10:29 IST" name="testForNull" finished-at="2022-12-07T13:10:29 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForNull -->
        <test-method signature="testForNotNull()[pri:6, instance:TestTopics.AssertsTopics@6aba2b86]" started-at="2022-12-07T13:10:29 IST" name="testForNotNull" finished-at="2022-12-07T13:10:29 IST" duration-ms="6" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForNotNull -->
        <test-method signature="testForHardAsserts()[pri:7, instance:TestTopics.AssertsTopics@6aba2b86]" started-at="2022-12-07T13:10:29 IST" name="testForHardAsserts" finished-at="2022-12-07T13:10:31 IST" duration-ms="2064" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [null] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [null] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotSame(Assert.java:1033)
at org.testng.Assert.assertNull(Assert.java:965)
at org.testng.Assert.assertNull(Assert.java:953)
at TestTopics.AssertsTopics.testForHardAsserts(AssertsTopics.java:87)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForHardAsserts -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:TestTopics.AssertsTopics@6aba2b86]" started-at="2022-12-07T13:10:31 IST" name="teardown" finished-at="2022-12-07T13:10:32 IST" duration-ms="130" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- TestTopics.AssertsTopics -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
